docs/csharp/advanced-topics/performance/index.md

```diff
@@ -1,8 +1,7 @@
 ---
 title: Avoid memory allocations and data copies
 description: Performance work in .NET means removing allocations from your code. One technique is to change critical data structures from `class` to `struct`. That changes semantics and means more data is being copied. Learn how to minimize allocations while preserving semantics and avoid extra copies.
-ms.date: 02/03/2023
-ms.technology: csharp-advanced-concepts
```

Comment:
I removed this because it's already set to the same value in the docfx.json file.